common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' title='Close modal'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
+        "  <button class='close-modal dismiss' title='" + gettext("Close") + "'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
@cahrens: I know you recommended moving safe template work to a separate PR, but we should probably avoid adding new errors.  

Generally, we want to avoid concatenation with HTML, because it is too easy to add more issues like this one, where it is now translated, but is not escaped.  Should this whole big string be moved to a separate .underscore file?  Or, do you want to use HtmlUtils calls, which will get a little messy.